### PR TITLE
Add Tre'bivdil to MOULa

### PR DESCRIPTION
**Pull request by DoobesURU**
Add Tre'bivdil .age and .fni files

Age Details:

* Name: Tre'bivdil
* Filename: trebivdil
* Creator: Lontahv
* Prefix: 10228

### CODE REVIEW

File: Scripts/dat/trebivdil.fni [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Graphics.Renderer.SetClearColor 0.5 0.699999988079071 0.800000011920929
```
Comment (zrax) [inline Scripts/dat/trebivdil.fni:1]: Are these numbers intentionally specific, or just IEEE 754 rounding errors for 0.7 and 0.8?

Comment (Deledrius) [inline Scripts/dat/trebivdil.fni:1]:
```suggestion
Graphics.Renderer.SetClearColor 0.5 0.7 0.8
```
These are the values in the original .blend.

File: Scripts/dat/trebivdil.fni [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+Graphics.Renderer.SetYon 100000
+Graphics.Renderer.Fog.SetDefColor 0.5 0.699999988079071 0.800000011920929
+Graphics.Renderer.Fog.SetDefLinear 0.0 0.0 0.0
```
Comment (Deledrius) [inline Scripts/dat/trebivdil.fni:4]:
```suggestion
Graphics.Renderer.Fog.SetDefColor 0.5 0.7 0.8
Graphics.Renderer.Fog.SetDefLinear 1 1000 0
```
Use values from original .blend.

File: Scripts/dat/trebivdil.fni (hunk 1 above)
Comment (DoobesURU) [reply Scripts/dat/trebivdil.fni:1]: They're entered as 0.7 and 0.8 in Korman, but export that way. /shrug

**Comment by DoobesURU** [comment]
Committed Del's changes.

File: Scripts/SDL/trebivdil.sdl [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+#       Mead, WA   99021
+#
+# *==LICENSE==*/
+
+STATEDESC trebivdil
+{
+    VERSION 1
+    
+    VAR BOOL trebChisoBookVis[1]        DEFAULT=1    DEFAULTOPTION=VAULT
+
```
Comment (Hoikas) [inline Scripts/SDL/trebivdil.sdl:48]:
```suggestion
```

(removes extra blank line)

File: Scripts/Python/trebivdil.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+#  or by snail mail at:
+#       Cyan Worlds, Inc.
+#       14617 N Newport Hwy
+#       Mead, WA   99021
+#
+# *==LICENSE==*/
+
+from Plasma import *
+from PlasmaTypes import *
+class trebivdil(ptResponder):
```
Comment (Hoikas) [inline Scripts/Python/trebivdil.py:45]:
```suggestion

class trebivdil(ptResponder):
```

(inserts blank line between imports and class)

File: Scripts/Python/trebivdil.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    def __init__(self):
+        ptResponder.__init__(self)
+        self.id = -1
+        self.version = 0
```
Comment (Hoikas) [inline Scripts/Python/trebivdil.py:49]:
```suggestion
        self.version = 0

```

(add blank line at end of file)

**Pull request merged**